.github/workflows/build_workflow.yml

```diff
@@ -64,11 +64,13 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2
         with:
           activate-environment: zppy_dev
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          use-mamba: true
+          mamba-version: "*"
           environment-file: conda/dev.yml
           channel-priority: strict
           auto-update-conda: true
-          # IMPORTANT: This needs to be set for caching to work properly!
-          use-only-tar-bz2: true
```

Comment:
I don't think this is necessary any longer.

Reply:
And I think it causes problems because most newer releases are as `.conda` files.

Reply:
This may explain why `sphinx-multiversion` couldn't be installed.